types/onfleet__node-onfleet/Resources/Tasks.d.ts

```diff
@@ -164,6 +163,21 @@ declare namespace Task {
     eta: number;
     trackingViewed: boolean;
   }
+
+  interface WorkerTaskContainer {
+    type: 'WORKER';
+    worker: string;
+  }
```

Comment:
nit: let's add a line break after the container interfaces 
```suggestion
  }

```